source/_posts/2020-11-12-android-300-release.markdown

```diff
@@ -0,0 +1,128 @@
+
+## Onboarding Improvements
+
+The first big change is that our onboarding screen has changed, we now ask the user to enter their preferred device name that gets attached to all entities that are created by the app. This was a source of frustration for some users as this meant that anytime they logged into the app they would then need to rename their entities to the name that they actually want. By default the device name is the device model but that is not meaningful to some users, especially if they have more than one device of the same model.
+
+
+<p class='img'>
+<img src='/images/blog/2020-11-05-android-300-release/onboarding.png' alt='Screenshot of new onboarding flow'></a>
+Screenshot of new onboarding flow.
+</p>
+
+
+## New Sensors
+
+We have several new sensors to welcome to the app, all of which are disabled by default. The first set of sensors were actually introduced in 2.5.0:
+
+- 4 Traffic Stat sensors - Sensors whose state represent the amount of data sent and received by the device. Mobile data may not be accurate, it depends on the data tat we get from the API.
+
+The following sensors are new in 3.0.0:
+
+- 4 Keyguard sensors - Sensors that represent various states from the Keyguard API, such as if a device is currently locked or has a password setup. These sensors will update during the periodic 15 minute interval.
+- Last Notification sensor - A very powerful sensor that requires a special permission you must grant to read all notifications posted on the device. All attributes of the notification are provided as attributes for the sensor. You can think of this sensor as a great way to integrate any app that posts a notification to your device allowing you to automate around it. Personally I have been using it to help integrate apps that don't integrate with Home Assistant Core a food delivery app for example, you can tell when your food has been delivered and automate around it. This sensor will update as soon as a notification is posted.
+- 3 Sleep as Android sensors - Users who have the [Sleep as Android](https://play.google.com/store/apps/details?id=com.urbandroid.sleep&hl=en_US) app installed can have sensors added that will represent the state of various events from the app. One big benefit this has is that you can tell when an alarm is currently active, snoozed, dismissed or if you are sleeping. This helps solve a lot of issues users have been having around trying to automate the Next Alarm sensor, which is a timestamp sensor and usually requires some sort of [templating](https://community.home-assistant.io/t/how-to-use-next-alarm-sensor/221686). These sensors all update immediately upon a state change.
```

Comment:
I think these look odd with the number at the start. Maybe drop them? It should be easy to tell how many there are from reading the items

Reply:
Will remove and link to the docs for each of these :)